ProjectSettings/ProjectSettings.asset

```diff
@@ -13,7 +13,7 @@ PlayerSettings:
   useOnDemandResources: 0
   accelerometerFrequency: 60
   companyName: DefaultCompany
-  productName: Runeshaper
+  productName: Runeshaper 2D
```

Comment:
Do you know if making this change would cause conflicts with people's local repositories?

Reply:
Unsure, changing it to "Runeshaper" just in case